docs/core/compatibility/sdk/6.0/install-location.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Breaking change: .NET can't be installed to custom location"
+description: Learn about the breaking change in .NET 6 where .NET can no longer be installed to a custom location using the MSI installer on Windows.
+ms.date: 08/02/2023
+---
+# .NET can't be installed to custom location
+
+You can no longer change the installation path of .NET with the Windows Installer package. To install .NET to a different path, use the [dotnet-install scripts](../../../tools/dotnet-install-script.md).
+
+## Version introduced
+
+.NET 6
+
+## Old behavior
+
+Previously, you could set `DOTNET_HOME` prior to running the windows MSI installer to install to a location other than *Program Files\dotnet*.
+
+## New behavior
+
+Starting in .NET 6, `DOTNET_HOME` is ignored and the SDK and runtime will always install under *Program Files\dotnet*. This impacts all .NET installers, including all three runtimes, the hosting bundle, and the SDK installer. It also impacts all architectures, even though the driver of the change was ARM64 support.
```

Comment:
Since this breaking change is about Windows only, I just clarified that and I don't think it makes sense to add the locations for macOS and Linux here, since they're not affected.